lib/cartopy/feature/nightshade.py

```diff
@@ -29,6 +29,11 @@ def __init__(self, date=None, delta=0.1, refraction=-0.83,
         refraction : float
             The adjustment in degrees due to refraction,
             thickness of the solar disc, elevation etc...
+        sub_solar_point : tuple of latitude, longitude
+            The coordinates in degrees of the point at which the sun is directly 
+            overhead. If both sub_solar_point and date are provided, sub_solar_point
+            takes precedence.
+            
```

Comment:
Extra blank line snuck in here.

Reply:
I have removed the extra whitespace, thanks.